CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Ensure adjacent rules are merged together after handling nesting when generating optimized CSS ([#14873](https://github.com/tailwindlabs/tailwindcss/pull/14873))
 - _Upgrade (experimental)_: Install `@tailwindcss/postcss` next to `tailwindcss` ([#14830](https://github.com/tailwindlabs/tailwindcss/pull/14830))
 - _Upgrade (experimental)_: Remove whitespace around `,` separator when print arbitrary values ([#14838](https://github.com/tailwindlabs/tailwindcss/pull/14838))
+- _Upgrade (experimental)_: Sort upgraded CSS ([#14866](https://github.com/tailwindlabs/tailwindcss/pull/14866))
```

Comment:
The current message makes it sound like the order was wrong or something but the change is only really for esthetic reasons, what do you think about this? I tried to not use the word order since it has an inherent meaning in CSS and am trying to lean on the "restructure" terminology

```suggestion
- _Upgrade (experimental)_: Restructure the generated CSS to make it easier to read ([#14866](https://github.com/tailwindlabs/tailwindcss/pull/14866))
```

Also this doesn't really fix anything so maybe it shouldn't be in the fixed category either I think 🤔 

I am still not convinced this is a very good message, maybe maybe this just doesn't need a changelog entry at all haha

Reply:
Yeah hmm, maybe this is just an internal refactor. Especially because the codemods are not idempotent right now meaning you only will be able to run them once anyway. So it's still a single 'upgrade' and not a fix to previous upgrades from the user's perspective.